src/sparse/KokkosSparse_Utils.hpp

```diff
@@ -715,7 +719,8 @@ void kk_create_reverse_map(
 
     // kk_inclusive_parallel_prefix_sum<reverse_array_type,
     // MyExecSpace>(tmp_reverse_size + 1, tmp_color_xadj);
-    kk_exclusive_parallel_prefix_sum<reverse_array_type, MyExecSpace>(
+    KokkosKernels::Impl::kk_exclusive_parallel_prefix_sum<reverse_array_type,
```

Comment:
Same suggestion as above. Remove `using KokkosKernels::Impl` or switch to `KkI::Impl`?